api/jobsendpoint.go

```diff
@@ -0,0 +1,100 @@
+package api
+
+import (
+	"fmt"
+	"github.com/cyverse-de/de-stats/cron"
+	"github.com/cyverse-de/de-stats/util"
+	"github.com/labstack/echo"
+	"net/http"
+	"time"
+)
+
+type JobsParams struct {
+	//The beginning of the time period of the response
+	//
+	//in: query
+	//required: false
+	//default: one week ago
+	StartDate string
+
+	//The end of the time period of the response
+	//
+	//in: query
+	//required: false
+	//default: today
+	EndDate string
+}
+
+type JobsResponse struct {
+	Count int	 	`json:"count"`
+	Jobs []cron.Job `json:"jobs"`
+}
+
+func verifyJobsParameters(ctx echo.Context) (string, string, error) {
+	const (
+		dateFormat = "20060102"
+	)
+
+	currentTime := time.Now()
+	oneWeekAgo := time.Now().AddDate(0, 0, -7)
+
+	startDate, err := util.StringQueryParam(ctx, "startDate", oneWeekAgo.Format(dateFormat))
+	fmt.Println(startDate)
+	if err != nil {
+		return "", "", ctx.JSON(http.StatusBadRequest, ErrorResponse{Description: err.Error()})
+	}
+
+	endDate, err := util.StringQueryParam(ctx, "endDate", currentTime.Format(dateFormat))
```

Comment:
I notice you're not adding a day to this end date (see my other comment about that). Definitely should be consistent about the semantics of the end date arguments, so this code should probably also know to add a day (or, as I recommended in the other comment, the database code could handle it).